### ISSUE TITLE
Permit AppAssertionCredentials to specify the service account to use

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#325]
```
This will let oauth2client.appengine.AppAssertionCredentials work for the 
(rare) case in which the current context cannot supply the appropriate service 
account id.
```

Original issue reported on code.google.com by `roh...@google.com` on 4 Feb 2014 at 10:21

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 54dc473e19f6.
```

Original comment by `jcgregorio@google.com` on 5 Feb 2014 at 7:27
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Passing a service_account_id using a the service account email seen in the 
developer console doesn't work. It produces a AccessTokenRefreshError when 
executing a request. Am I using it wrong?

More details here:
http://stackoverflow.com/questions/24327924/cloud-storage-api-requests-from-gae-
403-access-not-configured/24588927#24588927
```

Original comment by `romain.d...@gmail.com` on 5 Jul 2014 at 6:20
